packages/w/waline.json

```
@@ -20,10 +20,11 @@
       {
         "basePath": "dist",
         "files": [
-          "*.js"
+          "waline.css",
+          "waline.js",
+          "pageview.js"
```

```suggestion
          "*.@(js|css)"
```
It looks like there are quite a few other JS files beyond just `waline.js` and `pageview.js` that other developers may want to us, so let's select every JS/CSS file